CMakeLists.txt

```diff
@@ -255,7 +255,7 @@ if(TESTS)
 endif()
 
 # installation
-set_target_properties(zilliqa sendcmd genkeypair getpub getaddr gentxn signmultisig verifymultisig ZilliqaDaemon_AWS gensigninitialds grepperf getnetworkhistory getrewardhistory validateDB restore genTxnBodiesFromS3
+set_target_properties(zilliqa sendcmd genkeypair getpub getaddr gentxn signmultisig verifymultisig ZilliqaDaemon gensigninitialds grepperf getnetworkhistory getrewardhistory validateDB restore genTxnBodiesFromS3
```

Comment:
Can we change the daemon binary name (`ZilliqaDaemon`) to a name consisting of lowercase letters? Following the typical naming convention for daemon, a better name is `zilliqad`